docs/index.md

```diff
@@ -9,8 +9,8 @@ under neighborhoods throughout the city and counties. Some pipes are 100 years o
 pipes are decayed and leaking, and nearly a thousand gas leaks require fixes by Richmond's Department of Public 
 Utilities. Dozens of these leaks are liable to ignite or explode. 
 
-`INSERT LEAK MAP`
-
+{% include leak-map.html %}
+\
```

Comment:
This backslash is a markdown line break which helps some of the padding below the map.

Reply:
Good to know